index.md

```diff
@@ -148,6 +148,7 @@ customization safe and easy.
 
 ### Tutorials
 
+* Abdullah Khan wrote a [How to manage dotfiles securely](https://theak.me/blog/how-to-manage-dotfiles.html) on how you can encrypt your dotfiles using your gpg keys.
```

Comment:
I'd reword this either to "wrote an article" or "wrote" instead of "wrote a". Otherwise this looks good.